.golangci.yaml

```diff
@@ -4,3 +4,10 @@ run:
     - ".*\\.pb\\.go$"
     - "pkg/assembler/generated/.*"
     - "resolvers/schema\\.resolvers\\.go"
+linters:
+  enable:
+    - wrapcheck
+issues:
+  # Exclude previously existing issues from the report
```

Comment:
I think we want to include the existing issues and a PR to fix them. Can make it part of this PR.

Reply:
50 checks need to be fixed, and I can take that up in a following PR.

Reply:
I just realized that it is a lot more than 50 checks! :)